lib/phoenix/live_dashboard/live/processes_live.ex

```diff
@@ -125,6 +125,7 @@ defmodule Phoenix.LiveDashboard.ProcessesLive do
   @impl true
   def handle_event("search", %{"search" => search}, socket) do
     %{menu: menu, params: params} = socket.assigns
+    search = String.downcase(search)
```

Comment:
If we do this here, it means we will rewrite the input after the user submits it? Maybe it is best to downcsce it only before calling SystemInfo (or inside SystemInfo but o  the caller side).

Reply:
Yes, good call, can you check the latest? Or you meant to downcast it _before_ :rcp.call?